video/out/vo.c

```diff
@@ -87,6 +88,9 @@ const struct vo_driver *const video_out_drivers[] =
 #if HAVE_WAYLAND && HAVE_MEMFD_CREATE
     &video_out_wlshm,
 #endif
+#if HAVE_WAYLAND && HAVE_DRM && HAVE_MEMFD_CREATE
+    &video_out_dmabuf_wayland,
+#endif
```

Comment:
Can you place this one right above where `vaapi_wayland` is in the autoprobe order?